spec/services/hyrax/statistics/collections/over_time_spec.rb

```diff
@@ -5,10 +5,24 @@
   describe "#points", :clean_repo do
     before { FactoryBot.valkyrie_create(:hyrax_collection) }
 
-    it "is a list of points" do
-      expect(service.points.size).to eq 5
-      expect(service.points.first[1]).to eq 0
-      expect(service.points.to_a.last[1]).to eq 1
+    shared_examples('test list of points') do
+      it "is a list of points" do
+        expect(service.points.size).to eq 5
+        expect(service.points.first[1]).to eq 0
+        expect(service.points.to_a.last[1]).to eq 1
+      end
+    end
+
+    context 'ActiveFedora environment', :active_fedora do
+      include_examples 'test list of points'
+    end
+
+    # NOTE: Hyrax::AbstractTypeRelation inherits from ActiveFedora::Relation. When the code attempts to run
+    #   `AbstractTypeRelation.new(allowable_types: [Hyrax.config.collection_class])` in a Valkyrie environment,
+    #   it quietly fails with `#<NoMethodError: undefined method `solr_query_handler' for Monograph:Class> rescued
+    #   during inspection` and returns nil.
+    context 'Valkyrie environment' do
+      # include_examples 'test list of points'
     end
```

Comment:
Let's keep the failure more visible.
```suggestion
    context 'Valkyrie environment', pending: 'undefined method solr_query_handler' do
      include_examples 'test list of points'
    end
```

Reply:
My next commit will have this, plus a conditional `if Hyrax.config.disable_wings` on the Valkyrie tests so that it passes on the Dassie side.